python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -26,29 +30,44 @@ def betweenness_centrality(G, k=None, normalized=True,
     G : cuGraph.Graph
         cuGraph graph descriptor with connectivity information. The graph can
         be either directed (DiGraph) or undirected (Graph)
-    k : int, optional
-        Default is None.
+
+    k : int or list or None, optional, default=None
         If k is not None, use k node samples to estimate betweenness.  Higher
         values give better approximation
+        If k is a list, use the content of the list for estimation
+
     normalized : bool, optional
         Default is True.
         If true, the betweenness values are normalized by
         2/((n-1)(n-2)) for Graphs (undirected), and
         1 / ((n-1)(n-2)) for DiGraphs (directed graphs)
         where n is the number of nodes in G.
-    weight : cudf.Series
+
+    weight : cudf.Series, optional, default=None
         Specifies the weights to be used for each vertex.
-    endpoints : bool, optional
+
+    endpoints : bool, optional, default=False
         If true, include the endpoints in the shortest path counts
+
+    implementation : string, optional, default=None
+        if implementation is None or "default", uses native cugraph,
```

Comment:
If we need to keep both, we should advise on when to use one or the other and set a default (it seems the code does but the doc says something different).

Reply:
Updated in 50b27a1